packages/playground/docs/landing-page.html

```diff
@@ -56,6 +52,7 @@
             <section class="welcome-section">
                 <h1 class="title">UI5&nbsp;<span class="orange">Web Components</span></h1>
                 <div class="description">Enterprise-flavored sugar on top of native APIs!</div>
+                <p class="version">Latest Version: <a href="https://github.com/SAP/ui5-webcomponents/releases/tag/v1.9.0">1.9.0</a></p>
```

Comment:
the link does not point to existing page, we don't have 1.9.0, let's change it 1.8.0 and merge it